test/spec/modules/vidoomyBidAdapter_spec.js

```diff
@@ -0,0 +1,209 @@
+import { expect } from 'chai';
+import { spec } from 'modules/vidoomyBidAdapter.js';
+import { newBidder } from 'src/adapters/bidderFactory.js';
+
+const ENDPOINT = `https://d.vidoomy.com/api/rtbserver/prebid/`;
+
+describe('vidoomyBidAdapter', function() {
+  const adapter = newBidder(spec);
+
+  describe('isBidRequestValid', function () {
+    let bid;
+    beforeEach(() => {
+      bid = {
+        'bidder': 'vidoomy',
+        'params': {
+          pid: '123123',
+          id: '123123'
+        },
+        'adUnitCode': 'code',
+        'sizes': [[300, 250]]
+      };
+    });
+
+    it('should return true when required params found', function () {
+      expect(spec.isBidRequestValid(bid)).to.equal(true);
+    });
+
+    it('should return false when pid is empty', function () {
+      bid.params.pid = '';
+      expect(spec.isBidRequestValid(bid)).to.equal(false);
+    });
+
+    it('should return false when id is empty', function () {
+      bid.params.id = '';
+      expect(spec.isBidRequestValid(bid)).to.equal(false);
+    });
+
+    it('should return false when required params are not passed', function () {
+      let bid = Object.assign({}, bid);
+      bid.params = {};
+      expect(spec.isBidRequestValid(bid)).to.equal(false);
+    });
+
+    it('should return false when mediaType is video with INSTREAM context and lacks playerSize property', function () {
+      bid.params.mediaTypes = {
+        video: {
+          context: 'INSTREAM'
```

Comment:
This test is failing, to fix could you import INSTREAM from src/video and use here